### PR TITLE
Milestone 4: Calculator-Logic

### PR DESCRIPTION
**Pull request by LamiaSristy**
Here I have implemented the logical part in my calculator:
* Added a folder named Logic in the src folder.
* Added operate.js and calculate.js files.
* Added the big.js in my operate.js and added functionalities.
* Implemented the operations calculate.js.

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    if (operation === 'X') {
+      result = Big(num1.times(num2));
+    }
+
+    if (operation === '÷') {
+      result = Big(num1.div(num2));
+    }
+
+    if (operation === '%') {
```
Comment (mosaaleb) [inline src/logic/operate.js:25]: - [x] `%` in our case means turning a number into a percentage e.g. `95 % == .95`.

**Review by mosaaleb — CHANGES_REQUESTED** [review]
#### Changes required

Excellent job @LamiaSristy :clap:. You have handled all the requirements for this milestone. However, there is one requirement missing. Kindly check my comment below.


Please handle this requirement and submit a new code review.

##### Muhammad Ebeid

File: src/logic/operate.js (hunk 1 above)
Comment (LamiaSristy) [reply src/logic/operate.js:25]: Made the change you wanted.

**Review by mosaaleb — APPROVED** [review]
#### Milestone Approved :heavy_check_mark: 

Thanks for implementing the required changes. You can merge :+1: 
Good luck with the next milestones.

##### Muhammad Ebeid

**Pull request merged**